internal/levee.go

```diff
@@ -18,12 +18,14 @@ import (
 	"fmt"
```

Comment:
Sorry, for taking so long to come back to this PR.
I assumed that Patrick would complete the review. In the future, please ping me directly if you are not getting your work reviewed within a day.

Reply:
No worries at all. Thank you for the detailed review! This is exactly the kind of feedback I need to make progress in my ability to write idiomatic Go.